### PR TITLE
Fix deployment scripts

### PR DESCRIPTION
**Pull request by ShadoySV**

### CODE REVIEW

File: scripts/chain_configs/localterra.json [hunk 1]
```diff
@@ -1,54 +1,25 @@
...
+      "max_allocations_amount": "0"
     },
     "label": "Astroport Builder Unlocking Contract",
     "change_owner": false,
     "propose_new_owner": {
-      "owner": "terra17lmam6zguazs5q5u6z5mmx76uj63gldnse2pdp",
+      "owner": "",
       "expires_in": 604800
     },
-    "setup_allocations": {
```
Comment (P-Yevhenii) [inline scripts/chain_configs/localterra.json:22]: Why you remove allocations for Localterra?

**Review by P-Yevhenii — APPROVED** [review]

File: scripts/chain_configs/localterra.json (hunk 1 above)
Comment (ShadoySV) [reply scripts/chain_configs/localterra.json:22]: Because we don't have these addresses upon a new LocalTerra instance is created

Comment (epanchee) [reply scripts/chain_configs/localterra.json:22]: We need to have at least one allocation example

Comment (ShadoySV) [reply scripts/chain_configs/localterra.json:22]: Added

**Review by epanchee — APPROVED** [review]

File: scripts/chain_configs/localterra.json [hunk 2]
```diff
@@ -19,7 +19,18 @@
       "owner": "",
       "expires_in": 604800
     },
-    "allocations": []
+    "allocations": [
+      "terra1x46rqay4d3cssq8gxxvqz8xt6nwlz4td20k38v",
+      {
+        "amount": "1000000000",
```
Comment (P-Yevhenii) [inline scripts/chain_configs/localterra.json:25]: [here](https://github.com/astroport-fi/astroport-governance/blob/bec6609364cc0ac7f46092c7bf017f1574711b42/scripts/deploy.ts#L183) we will get an error. We should set max_allocations_amount to sum of allocations.

**Review by P-Yevhenii — APPROVED** [review]

**Review by P-Yevhenii — APPROVED** [review]

**Review by epanchee — APPROVED** [review]

**Pull request merged**